src/component/panels/SummaryPanel/CorrelationTable.jsx

```diff
@@ -50,7 +51,7 @@ const getExperiment = (data) => {
   }
 
   pulse = pulse.toLowerCase();
-  if (pulse.includes('zg')) {
+  if (pulse.includes('zg') || pulse.includes('udeft')) {
     // + udeft ?
     return '1d';
   }
```

Comment:
I have an idea to remove all the if statement and reduce the code

I think you can use regular expression directly inside of includes, the idea is to have a table containing return value as key and expression( regular expression represent every if statement) and loop throw it row by row, and if the condition is true return the key and break the loop... 

this change not mandatory but you have to move all the if statement to function and then call the function here